### PR TITLE
Add the privacy policy for the Opf DS Community Meetup

### LINKED ISSUE #28: Create a Policy for the Opf DS Community Meetup
We would want to create a policy for the Opf DS Community Meetup, that would allow the users to know what all data we would be recording like the meetups being recorded, the attendees data being recorded. 

We would also however ensure that the data is not posted publicly with the attendees' public information.

Acceptance Criteria - 
Policy document for Operate First Data Science Community Meetup

### PR DESCRIPTION
**Pull request by aakankshaduggal**
This PR resolves #28

### CODE REVIEW

**Comment by aakankshaduggal** [comment]
@quaid : We have added a policy for our Operate First Data Science Community Meetup in this PR since we would be tracking the number of attendees and using this data for calculation of some metrics as mentioned in #35 , recording the meetups and publishing them on our youtube channel.
Could you please help in reviewing this document, thank you :)

**Comment by erikerlandson** [comment]
Basically I think this has an excellent structure.  I think the one thing I would want to specify is WHAT information you collect (and possibly mention some things you WON'T collect, if some things are commonly considered and people will wonder about).

I think the list information you are planning to collect is relatively small.  Usernames, emails, meeting recordings, and associated meeting minutes.  These should be listed.

**Comment by erikerlandson** [comment]
Maybe have some point-of-contact where people can register a complaint/issue (privately) if they feel policy is violated somehow

File: docs/privacy-policy.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+## Collection Of Information
+
+We collect information about you when you:
+
+1. Login using a google account;
+2. Use the google meet platform;
+3. Attend an event;
+4. Visit our website; and
+5. Communicate with us.
```
Comment (suppathak) [inline docs/privacy-policy.md:30]: Shall we add the means of communication? is it emails, google chat, slack or more?

Comment (MichaelClifford) [reply docs/privacy-policy.md:30]: Yes, "communicate with us" is vague. What does it mean?

File: docs/privacy-policy.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+We may also collect information, such as usage statistics, by using cookies, server logs, and similar technologies.
+
+
+## Use Of Information
+
+We use information about you to:
+
+1. Operate and promote our Community;
```
Comment (MichaelClifford) [inline docs/privacy-policy.md:39]: What Does "operate" mean here? We use it to promote upcoming events, but I'm not sure about "operate"

File: docs/privacy-policy.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+3. Communicate with you; and
+4. Improve content.
+
+
+## Sharing Of Information
+
+We do not share information about you except as described in this Privacy Policy. We may disclose personal information about you, including:
+
+1. To authorities to comply with legal obligations or requests and to protect and defend our rights and property;
+2. To visualize key performance indicator metrics such as attendees over time, distribution of organizations represented in audience, attendees vs topic presented, average time duration of the meeting, etc. For calculating such KPIs, we anonymized user data, and store the user to anonymous id (e.g. SHA) mapping somewhere secure where very limited people have access.
```
Comment (MichaelClifford) [inline docs/privacy-policy.md:50]:
```suggestion
2. To visualize key performance indicator metrics such as attendees over time, distribution of organizations represented in audience, attendees vs topic presented, average time duration of the meeting, etc. For calculating such KPIs, we anonymized user data, and store the anonymous id (e.g. SHA) mapping somewhere secure where very limited people have access.
```

Why do we need to keep the SHA mapping?

File: docs/privacy-policy.md (hunk 1 above)
Comment (Shreyanand) [reply docs/privacy-policy.md:30]: The list structure here can be confusing. We don't collect information when someone logs in to their gmail or uses the google meet platform. We only collect information when they attend the DS meetup event. 
So maybe just three points
1) Attend DS community meetup through google meet platform
2) Visit our website
3) Communicate using ....
WDYT?

File: docs/privacy-policy.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+We collect information about you when you:
+
+1. Login using a google account;
+2. Use the google meet platform;
+3. Attend an event;
+4. Visit our website; and
+5. Communicate with us.
+
+We may also collect information, such as usage statistics, by using cookies, server logs, and similar technologies.
```
Comment (Shreyanand) [inline docs/privacy-policy.md:32]: Should we leave this out until we actually use and track these statistics? And update the doc with exact stats that we collect once we do?

**Review by Shreyanand — COMMENTED** [review]
Great doc :clap: Added some comments :)

File: docs/privacy-policy.md (hunk 1 above)
Comment (aakankshaduggal) [reply docs/privacy-policy.md]: Sounds good, we have rephrased this section :+1:

**Comment by aakankshaduggal** [comment]
Thank you all for your review, we have made the changes. :+1:

File: docs/privacy-policy.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+5. [References](#References)
+
+6. [Contact Us](#Contact-Us)
+
+
+
+## Collection Of Information
+
+We use Google Meet to host the meetups. Through Google Meet, we get the following information about you, when you attend when you attend the Operate First Data Science Community Meetups:
```
Comment (suppathak) [inline docs/privacy-policy.md:24]: Just a small typo:
```suggestion
We use Google Meet to host the meetups. Through Google Meet, we get the following information about you, when you attend the Operate First Data Science Community Meetups:
```

File: docs/privacy-policy.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+    *b.* as you provide in the “Your name” section, in case you join anonymously
+
+2. Email ID or phone number you used to join the event
+3. Timestamp(s) when you joined the event
+4. Timestamp(s) when you left the event
+
+We use Google Analytics on the operate-first website. This means that, through Google Analytics, we get the following information about you, when you visit our website:
+Where you got to the page from (e.g. mail,github, twitter, youtube, etc.)
+We do **not** get any personal information through this.
```
Comment (suppathak) [inline docs/privacy-policy.md:37]: I guess these two points should be like a bullet points right? If so, please reformat the text. They are showing up in a single line. 
```suggestion
Where you got to the page from (e.g. mail,github, twitter, youtube, etc.)
We do **not** get any personal information through this.
```

File: docs/privacy-policy.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+We use information about you to:
+
+1. Promote our Community;
+2. Track engagement;
+3. Communicate with you; and
+4. Improve content.
+
+## Sharing Of Information
+
+We do not share information about you except as described in this Privacy Policy. We may disclose personal information about you, including:
```
Comment (suppathak) [inline docs/privacy-policy.md:54]: Just a thought on making it a bit concise. Lmk if it makes sense.
```suggestion
We do not share information about you except as described in this Privacy Policy below,
```

**Review by suppathak — COMMENTED** [review]
Jotted down some minor suggestions. :)

File: docs/privacy-policy.md (hunk 7 above)
Comment (aakankshaduggal) [reply docs/privacy-policy.md:55]: I think it's vital to mention that we might be using this personal information for various reasons as mentioned here.

File: docs/privacy-policy.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+    *a.* as it appears on your Google Account, in case you join through your Google Account; or
+
+    *b.* as you provide in the “Your name” section, in case you join anonymously
+
+2. Email ID or phone number you used to join the event
+3. Timestamp(s) when you joined the event
+4. Timestamp(s) when you left the event
+
+We use Google Analytics on the operate-first website. This means that, through Google Analytics, we get the following information about you, when you visit our website:
+Where you got to the page from (e.g. mail,github, twitter, youtube, etc.)
```
Comment (MichaelClifford) [inline docs/privacy-policy.md:36]: Do we actually get individual user info here, like email address? Or just "some unique user" went to N pages?

File: docs/privacy-policy.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+We do **not** get any personal information through this.
+
+We may also collect information about you such as your name and email ID, when you communicate with us via email or slack.
+
+In addition to this, we also collect visitor metrics and related information like where did you get to know about this page, whenever you visit the website.
```
Comment (MichaelClifford) [inline docs/privacy-policy.md:42]: This sounds like a repeat of "we get the following information about you, when you visit our website:" above. How is it different?

File: docs/privacy-policy.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+
```
Comment (MichaelClifford) [inline docs/privacy-policy.md:43]: We repeatedly say "we get information about *YOU*" but then note, "we do not get any personal information".  This is a little confusing to me, What actual identifying information do we get about users through google analytics?

**Review by MichaelClifford — COMMENTED** [review]
:+1:

File: docs/privacy-policy.md (hunk 8 above)
Comment (schwesig) [reply docs/privacy-policy.md:36]: +1

File: docs/privacy-policy.md (hunk 10 above)
Comment (schwesig) [reply docs/privacy-policy.md:43]: +1

File: docs/privacy-policy.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+    *a.* as it appears on your Google Account, in case you join through your Google Account; or
+
+    *b.* as you provide in the “Your name” section, in case you join anonymously
+
+2. Email ID or phone number you used to join the event
+3. Timestamp(s) when you joined the event
+4. Timestamp(s) when you left the event
+
+We use Google Analytics on the operate-first website. This means that, through Google Analytics, we get the following information about you, when you visit our website:
```
Comment (Shreyanand) [inline docs/privacy-policy.md:35]: Google analytics collects a lot of information such as location of the user, number of times they logged in, which page they spent the most time at etc., PTAL at a [google analytics privacy policy template.](https://www.termsfeed.com/public/uploads/2022/01/sample-google-analytics-privacy-policy-template.pdf) maybe we can borrow something from here.

**Review by Shreyanand — CHANGES_REQUESTED** [review]

File: docs/privacy-policy.md (hunk 8 above)
Comment (aakankshaduggal) [reply docs/privacy-policy.md:36]: It's mainly just the number of users, let me rephrase this to make it easier to understand.

File: docs/privacy-policy.md (hunk 9 above)
Comment (aakankshaduggal) [reply docs/privacy-policy.md:42]: Will remove this.

File: docs/privacy-policy.md (hunk 11 above)
Comment (aakankshaduggal) [reply docs/privacy-policy.md:35]: Thanks Shrey, will look into this :+1:

File: docs/privacy-policy.md [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+
+    *a.* as it appears on your Google Account, in case you join through your Google Account; or
+
+    *b.* as you provide in the “Your name” section, in case you join anonymously
+
+2. Email ID or phone number you used to join the event
+3. Timestamp(s) when you joined the event
+4. Timestamp(s) when you left the event
+
+We use **Google Analytics** on the operate-first website. This means that, through Google Analytics, we get information such a your Device's Internet Protocol address (e.g. IP address), browser type, browser version, the pages of our Service that You visit, the time and date of Your visit, the time spent on those pages, unique device identifiers and other diagnostic data. We do **not** get any personal information through google analytics.
```
Comment (suppathak) [inline docs/privacy-policy.md:35]:
```suggestion
We use **Google Analytics** on the operate-first website. This means that when you visit [operate-first](https://www.operate-first.cloud/) website, through Google Analytics, we get information such as your Device's Internet Protocol address (e.g. IP address), browser type, browser version, the pages of our Service that you visit, the time and date of your visit, the time spent on those pages, unique device identifiers and other diagnostic data. We do **not** get any personal information through google analytics.
```

```
 ..and other diagnostic data. We do **not** get any personal information through google analytics.
```

 The term `other diagnostic data` is making me think what other metrics could be there that is shared? Can we be more specific? wdyt? 

@aakankshaduggal , wonderful doc :clap: . I just added some comments and suggestions. Let me know if it makes sense to you. :) Thanks

**Review by schwesig — APPROVED** [review]
/lgtm
/approved

**Review by Shreyanand — COMMENTED** [review]
/lgtm :+1:

**Review by suppathak — COMMENTED** [review]
/lgtm 
Thanks @aakankshaduggal :)
